RocketChat/Rocket.Chat
# [FIX] Cancel button and success toast at Leave Team modal

## Proposed changes (including videos or screenshots)





## Issue(s)


## Steps to test or reproduce

- Try to leave a Team
- If Successfull, a toast message should appear.
- If there are no rooms to select, a "Cancel" button should appear instead  of "back".
## Further comments
